umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md

```diff
@@ -55,68 +53,65 @@ Benthos UMH offers some benefits, including:
 ### With the Management Console
 
 The easiest way to use Benthos UMH is to deploy it directly from the Management
-Console
+Console.
 
-{{% notice warning %}}
-Currently, only OPC-UA data sources can be configured from the Management
-Console. To use other data sources, you must deploy Benthos UMH in standalone
-mode or use [Node-RED](/docs/features/connectivity/node-red/).
-{{% /notice %}}
+After adding your network device or service, you can initialize the protocol
+converter. Simply click on the **Play** button next to the network device/service
+at the **Protocol Converters** tab.
+From there, you'll have two options to choose from when configuring the
+protocol converter:
 
-You first have to add a new connection to your OPC-UA server from the
-**Connection Management** tab.
+- **Universal Protocol Converter**: Opt for this choice if you need to configure
+  protocol converters for various supported protocols other than OPC-UA. This option
+  will prompt you to define the Benthos input and processor configuration in YAML format.
 
-![Add Connection](/images/features/data-connectivity-benthos/connection-management.png?width=80%)
+- **OPC-UA**: Select this option if you specifically need to configure
```

Comment:
Should the user now use OPC-UA or the Universal Protocol converter? What is the difference?

(in OPC UA, there is for example a better data contextualization part that allows it easier to assign tags to certain places in the UNS)